src/backend/app/s3.py

```diff
@@ -93,7 +93,7 @@ def get_file_from_bucket(bucket_name: str, s3_path: str, file_path: str):
     client.fget_object(bucket_name, s3_path, file_path)
 
 
-def get_obj_from_bucket(bucket_name: str, s3_path: str) -> BytesIO:
+def get_obj_from_bucket(bucket_name: str, s3_path: str) -> Optional[BytesIO]:
```

Comment:
Sorry to be pedantic, but now the error handling was reverted, this function always returns `BytesIO`, so `Optional` is not necessary.